app/controllers/supplementary/LocationController.scala

```diff
@@ -41,21 +42,20 @@ class LocationController @Inject()(
 
   def displayForm(): Action[AnyContent] = authenticate.async { implicit request =>
     customsCacheService.fetchAndGetEntry[GoodsLocation](appConfig.appName, formId).map {
-      case Some(data) => Ok(goods_location(appConfig, form.fill(data)))
-      case _          => Ok(goods_location(appConfig, form))
+      case Some(data) => Ok(goods_location(appConfig, form().fill(data)))
```

Comment:
You don't need to have `()` here. If you define a method as `def form()` you can invoke it with or without parenthesis. It's good to show that it's a method, but do we need a method for it?

It's correct but unnecessary in my opinion